### PR TITLE
eDoge token add request

### PR DESCRIPTION
**Pull request by OxGAB**

### CODE REVIEW

File: src/tokenlist.json [hunk 1]
```diff
@@ -135,5 +135,13 @@
       "address": "0x7e806d59528f6fa7cccadb4821dd42551113defc",
       "logoURI": "https://storage.googleapis.com/us-central1-dgc-berlin-0-470cbba9-bucket/tokenlist/DIFF.png"
     }
```
Comment (crypzoh) [inline src/tokenlist.json:137]: There is a `,` missing here

Comment (OxGAB) [reply src/tokenlist.json]: Hey! issue is resolved.

Comment (OxGAB) [reply src/tokenlist.json]: Hey there! I added the comma now.

**Comment by crypzoh** [comment]
I took your token info and added it, also fixed the link to the logo: https://github.com/diffusion-fi/tokenlist/pull/37

**Pull request closed without merging**